lib/tbot/workloadidentity/crl_cache.go

```diff
@@ -67,6 +69,52 @@ func (b *CRLSet) Stale() <-chan struct{} {
 	return b.stale
 }
 
+func NewCRLCacheFacade() *CRLCacheFacade {
```

Comment:
Is it worth having a short GoDoc here to explain that this Facade exists to provide lazy initialisation ?